packages/nextjs/src/components/index.ts

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import {
```

Comment:
Can't we export from @novu/react in a single file?

Reply:
The problem is we need to export everything from `@novu/react` except the `Inbox`
I tried few methods like ```
import * as NovuReact from'@novu/react
const {Inbox, ...rest} = NovuReact;
export {rest}
```
and other similar ways, they didn't worked :/